swap/src/bin/swap.rs

```diff
@@ -296,18 +298,18 @@ async fn init_bitcoin_wallet(
     .context("Failed to initialize Bitcoin wallet")?;
 
     let wallet_dir = data_dir.join("personal_wallet");
-    let _personal_wallet = bitcoin::wallet::Wallet::new_readonly(
+    let personal_wallet = bitcoin::Wallet::new_readonly(
```

Comment:
Having a "personal wallet" inside the CLI code that is not actually exposed to the user feels like unnecessary complexity to me. I don't want to need to read code that looks like it matter for the user but actually doesn't.